app/assets/styles/app.scss

```diff
@@ -1,6 +1,6 @@
 // Semantic UI breakpoints
 $mobile-breakpoint:        768px;
-$tablet-breakpoint:        992px;
+$tablet-breakpoint:        956px;
```

Comment:
That was the point where the last option (log in) was breaking into two lines because there is not enough space. But I think we can revert it back since we merged account into the dashboard